src/less/components/button.less

```diff
@@ -108,6 +110,7 @@
     &:focus {
         outline: 0;
         box-shadow: 0 0 0 0.1rem @focus-shadow;
+        border-color: @color;
     }
 
     &:hover,
```

Comment:
Probably not this line, but according to Figma, the hover color should be #FBDD91 and not #FFDF88.
The same applies to the focus state.